### PR TITLE
fix(youtube/integrations): set context for remaining activities

### PR DESCRIPTION
**Pull request by Canny1913**
Fixes video player in apps such as Play Store or Sync for Reddit.

### CODE REVIEW

**Review by oSumAtrIX — APPROVED** [review]
Looks good so far, I have done some refactoring which should be reviewed.

File: src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt [hunk 1]
```diff
@@ -30,49 +30,30 @@ class IntegrationsPatch : BytecodePatch(
...
-
-            method.addInstruction(
-                0,
-                "sput-object v$register, Lapp/revanced/integrations/utils/ReVancedUtils;->context:Landroid/content/Context;"
-            )
         }
 
-
-        val classDef = InitFingerprint.result!!.mutableClass
-        classDef.methods.add(
```
Comment (Canny1913) [inline src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: This function is used in integrations which should be removed.

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: This has already been removed in commit https://github.com/revanced/revanced-patches/pull/828/commits/5d8e2f5a947911bf17f412ec7d0d4f1f71139153.

Comment (Canny1913) [reply src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: https://github.com/revanced/revanced-integrations/blob/main/app/src/main/java/app/revanced/integrations/utils/ReVancedUtils.java#L64

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: Should the integrations not use [ReVancedUtils.getAppContext()](https://github.com/revanced/revanced-integrations/blob/main/app/src/main/java/app/revanced/integrations/utils/ReVancedUtils.java#L20) instead?

Comment (Canny1913) [reply src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: Both of functions are identical so yeah.

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt:45]: Can you open a PR for that?

**Pull request merged**